SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the SunPy software library
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. github account, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombusperson
+
+The SunPy project is ultimately run by the whole community: users, developers, and members of the wider solar physics and scientific software communtiy, for the benifit of all.
+This document descibes how the project is run through delegation of this authority, primarily to the Steering Committee and the Project Members.
+Wherever possible all descisions taken should be done by consensus, considering input from both active contributors and the wider community.
+
+## Advisory Board
+The primary functions of the advisory board are: to advise the steering committee on management of the project, find funding for the project to ensure the core aims of the project can be met, nominate a Ombudsperson, and to ensure that the governance of the project is running as intended.
+
+* **Membership Limit**: The board shall consist of at least 5 members and no more than 10 members.
+Within these limits, the board may increase or decrease the number of people serving on the board, including for the purpose of staggering the terms of members.
+
+* **Term Length**: All members shall be elected to serve a maximum of a two-year term, however the term may be extended until a successor has been elected. 
```

Comment:
I'd suggest not having extension of terms, to force the board to either conciously re-elect a member or find a replacement. This would help avoid 'zombie' boards where everyones term is extended in perpituity, which is what feels like has happened for the last few years...

Reply:
I have proposed an edit for this here: https://github.com/sunpy/sunpy-SEP/pull/62/files#r1144833596